slack-scripts/ocular-findings/manifest.yml

```diff
@@ -0,0 +1,28 @@
+_metadata:
+  major_version: 1
+  minor_version: 1
+display_information:
+  name: Ocular Findings
+features:
+  bot_user:
+    display_name: Ocular Findings
+    always_online: false
+oauth_config:
+  scopes:
+    bot:
+      - channels:join
+      - channels:read
+      - chat:write
+      - im:read
```

Comment:
Can we trim and make it work without im:read and user:read?

Reply:
I wanted it for some bot like features or user specific testing they might have wanted

Reply:
Removed them